test/data/migrations/pre-1.7.0/environments.json

```diff
@@ -12,6 +12,7 @@
         "endpoint": "*/:var/a(b)?c/[0-9]{1,5}",
         "documentation": "Test documentation",
         "enabled": true,
+        "randomResponse": false,
```

Comment:
Please do not modify files in `test/data/migrations`, `test/data/import/new` and `test/data/import/old`